benchmarks/README.md

```diff
@@ -2,7 +2,8 @@
 
 This suite implements the benchmarks described in this (spec)[https://github.com/mongodb/specifications/blob/master/source/benchmarking/benchmarking.rst].
 
-In order to run the microbenchmarks, first run `../etc/microbenchmark-test-data.sh` to download the data.
+In order to run the microbenchmarks, first run `./download-data.sh`. (NOTE: the data for the deeply nested BSON encoding and decoding is
+currently broken, so these benchmarks will not be runnable until that's fixed).
```

Comment:
in the meantime, we can just use the old version (pre-corruption) since it doesn't use any extJSON v2. We just checked it into the Swift benchmarking repo (it's really small: https://github.com/mongodb/mongo-swift-driver-performance/blob/main/data/deep_bson.json), so I think we can do the same here temporarily. I think it's probably important that we do these deep_bson benchmarks if we're planning on BSON being the biggest bottleneck in the driver right now.

Reply:
@saghm if you want you can also just do SPEC-1752 yourself. it should be really quick to replace the file and re-upload the fixed tarball, I just haven't been able to prioritize it. if not I can do it myself sometime this week

Reply:
Yep, I can do that. The only reason I didn't go ahead and do it before is that I couldn't actually find the correct file anywhere, but now that Patrick linked it, I'll make the PR to the specs repo.

Reply:
Once https://github.com/mongodb/specifications/pull/889 goes in can we update this accordingly?

Reply:
Definitely! I think it should probably be possible to merge that before this so we don't have to do a follow-up commit